src/components/README.md

```diff
@@ -1 +1 @@
-&#8203;
+&#8203;
```

Comment:
What's... the difference here?

Reply:
i don't know... i accidentally deleted the file then copy+pasted the text back